manuscript/draft.Rmd

```diff
@@ -283,83 +283,88 @@ biodiversity in the form of species richness. The results of this effort are
-show that for forecasts 1-3 years in the future, baselines and time-series based
-approaches outperform all other methods using integrative metrics. At these time
-scales environmental changes are relatively small and changes in the biota may
-lag behind changes in the environment. However, as the timescale of the forecast
-increases to a decade the JSDMs and environmental richness models get closer to
-the accuracy of the time-series methods and even surpass some of those
-methods. As the forecast distance increases the amount of environmental change
-is expected to increase and the system will have more time to respond,
-potentially leading to shifts towards improved relative performance of the
-models incorporating environmental and ecological information. If this shift
-with forecast distance in the relative importance of different processes and
-models continues, this could lead to JSDMs and environmental richness models
-outperforming time-series based approaches at sufficiently long time
-scales. These results suggest that assessment of how forecast performance
-changes with time lag is not only important for determining how far in the
-future to forecast, but also for determining models and processes are most
-relevant for making forecasts at different time scales.
+environmental variables and using the resulting models to make forecasts []. Two
+comparisons of the the SDM and direct richness modeling approaches reported that
+the methods yielded equivalent results for forecasting diversity (some other
+comparisons that found poor SDM performance were [[tainted]] by the rounding
+error alluded to above and are not considered here). While our results generally
+support rough equivalence between stacked SDMs and direct richness modeling for
+point estimates, they also show that modeling richness directly
+results in much better estimation of the uncertainty of the forecast and
+therefore is a better overall approach to forecasting richness. A similar result
+is seen when comparing joint species distribution models (JSDMs) to stacked
+single species distribution models. For point estimates the joint distribution
+models are roughly equivalent to stacked SDMs, but the JSDMs provide much better
+estimates of uncertainty. In fact, JSDMs and modeling richness directly provide
+some of the best estimates of uncertainty across all modeling approaches. This
+highlights the importance evaluating models based on their uncertainty as well
+as their mean estimates. Not doing so is particularly problematic in this case
+because the summed species distribution models are overly confident meaning that
+using them would lead to believing that richness would be restricted to a much
+narrower range than would actually occur.
+
+For error metrics that evaluate both models' point estimates and their
+uncertainty, the joint SDM and the nonlinear richness model began to outperform
+the time-series baselines at the longest time-scales assessed. This convergence
```

Comment:
Has something changed in the most recent Figure 3 results? The one in the slack discussion shows average and ARIMA still slightly outperforming richness_gbm and mistnet at the longest time scales.